server/datastore/mysql/migrations/tables/20201011162341_CleanupSoftDeletedColumns.go

```diff
@@ -0,0 +1,95 @@
+package tables
+
+import (
+	"fmt"
+
+	"database/sql"
+	"github.com/kolide/goose"
+)
+
+func init() {
+	goose.AddMigration(Up20201011162341, Down20201011162341)
```

Comment:
This needs to be `MigrationClient.AddMigration`. That will solve your issue with the migration not running. Do we need to update the docs about this?

Reply:
The docs say to run `goose create AddColumnFooToUsers`, then 'edit it', but is less clear on what needs to be done.

I see two things that **must** be edited (as far as I can tell) each time you run the `goose create` command:

1. change the import for goose from `github.com/pressly/goose` to `github.com/kolide/goose`.
2. change `goose.AddMigration(...)` to `MigrationClient.AddMigration(...)

---

For (1), I can add a note to the docs to change the import if you'd like.

It would also be good to have the `goose create` command generate this file as close to 'correctly' as possible, so it might be worth updating the goose library to do this.

Also while I was looking into this, I was a bit confused on what the kolide goose fork did, but it seemed like the fork hasn't been updated since 2018, while the pressly source is still actively maintained. Long-term - should we consider moving back to pressly upstream?

---

For (2), I can add a note to the docs about this if you'd like.

I also think it would be good to have the command generate this closer to 'correct' initially without requiring this change.